client/plugins/ip-port.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package conntrack
```

Comment:
should this be on `conntrack` folder, otherwise this is on plugins module.

Reply:
yeah, removing we dont need that file good catch thanks amim